drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -633,6 +653,13 @@ void nrf_802154_receive_failed(nrf_802154_rx_error_t error)
 	nrf5_data.last_frame_ack_fpb = false;
 }
 
+void nrf_802154_tx_started(const uint8_t * frame)
```

Comment:
no space between * and the parameter name

Reply:
fixed